src/test/java/com/org/jenkins/custom/jenkins/distribution/service/generators/WarGeneratorTest.java

```diff
@@ -1,12 +1,21 @@
 package com.org.jenkins.custom.jenkins.distribution.service.generators;
 
+import com.org.jenkins.custom.jenkins.distribution.service.util.Util;
 import org.junit.jupiter.api.Test;
 
+import static org.springframework.test.util.AssertionErrors.fail;
+
 public class WarGeneratorTest {
 
+    Util util = new Util();
+
     @Test
     public void testWarGenerationSucceeds() {
-        WarGenerator.generateWAR("1.0");
+        try {
+            WarGenerator.generateWAR("1.0", util.readStringFromFile("packager-config.yml") );
```

Comment:
Since you're catching the error message and just printing the statement, you'll never test if something went wrong.

Reply:
Changed the method to throw an exception